example/coreapi/core_api_credit_card_example.rb

```diff
@@ -37,23 +37,24 @@
   }
 )
 
-puts result.body
+puts result.data
```

Comment:
- Add example of handling/catching error
- Also don't just plainly print out variable. Add some descriptive text, e.g. print out "Charge result: "

Both of above are exist on the nodejs example as the reference, the example already have that, so that is the standard-quality, which means this example have less-quality than the reference. We should upgrade the quality of our work (learning from our past work), not downgrade.